docker-hub/billing/faq.md

```diff
@@ -214,338 +86,138 @@ For more information, view our blogs on [inactive image retention](https://www.d
   "mainEntity": [
     {
       "@type": "Question",
-      "name": "What plans and pricing changes did Docker announce on May 14?",
-      "acceptedAnswer": {
-        "@type": "Answer",
-        "text": "Docker announced the following plans and pricing changes.\n\nImmediately available for Individuals and Teams:\n\nFree plans will continue to be available for both individuals and development teams that include unlimited public repos.\nNEW Pro plan for individuals with unlimited private repos, unlimited public repos, and unlimited collaborators starting at $5/month with an Annual subscription.\nNEW Team plan for development teams with unlimited private repos and unlimited public repos starting at $25 per month for the first 5 users and $7 per month for each user thereafter with an Annual subscription. The Team plan offers advanced collaboration and management tools, including organization and team management with role-based access controls."
-      }
-    },
-    {
-      "@type": "Question",
-      "name": "Are prices listed in US dollars?",
-      "acceptedAnswer": {
-        "@type": "Answer",
-        "text": "All prices reflect US dollars."
-      }
-    },
-    {
-      "@type": "Question",
-      "name": "How does Team pricing work?",
+      "name": "What subscriptions levels are available for Docker?",
       "acceptedAnswer": {
         "@type": "Answer",
-        "text": "Team starts at $25 per month for the first 5 users and $7 per month for each user thereafter with an Annual subscription."
+        "text": "Docker Subscriptions come in three levels:\nFree plans for both individuals and development teams that include unlimited public repos.\nDocker Pro Subscriptions for individuals with unlimited private repos, unlimited public repos, no rate limiting on pull requests and premium support starting at $5/month with an annual subscription.\nDocker Team Subscriptions for development teams with unlimited private repos and unlimited public repos starting at $25 per month for the first 5 users and $7 per month for each user thereafter with an Annual subscription. The Team plan offers advanced collaboration and management tools, including organization and team management with role-based access controls.\n\n"
```

Comment:
Probably don't need the newlines at the end of this. Not an error, just a preference.

Reply:
Should I remove the \n from the lines?